editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -32,10 +32,12 @@
 
 #include "core/config/project_settings.h"
 #include "core/io/resource_loader.h"
+#include "core/io/resource_uid.h"
```

Comment:
Not needed, it's already included implicitly.

Reply:
Oops. Well, looks like this was added automatically by Cmantic.